build/jade.js

```diff
@@ -5,6 +5,9 @@ var find = require('find')
 var jade = require('jade')
 var async = require('async')
 var mkdirp = require('mkdirp')
+var extend = require('extend')
+var requireDir = require('require-dir')
+var helpers = requireDir('../helpers')
```

Comment:
I don't really understand this.  Surely the result of `path.join(__dirname, '..', 'helpers')` could just as well not exist?